src/main/java/org/ships/vessel/sign/MoveSign.java

```diff
@@ -36,9 +36,9 @@
 
     public final List<AText> SIGN = Arrays.asList(
             AText.ofPlain("[Move]").withColour(NamedTextColours.YELLOW),
-            AText.ofPlain("{Engine}}"),
             AText.ofPlain(""),
-            AText.ofPlain(2 + "")
+            AText.ofPlain("Speed"),
+            AText.ofPlain(1 + "")
```

Comment:
I do believe that initial speed 1 makes much more sense than 2 - feel free to revert it if you don't think so.

Reply:
Maybe it should be a value in config, let the admin decide